README.md

```diff
@@ -772,8 +776,128 @@ Note that the id that is passed to the `SerializationConfig` is same as the `fac
 
 ## 3. Custom Serialization
 
+Hazelcast lets you to plug a custom serializer to be used for serialization of objects.
+
+Let's say you have an object `Musician` and you would like to customize the serialization. The reason could be you are not happy with the default serialization.
```

Comment:
I think the most common reason for using a custom serializer is that the user wants to use an external serializer for only one object but let Hazelcast deal with the others. 

`The reason could be you are not happy with the default serialization.` I think this sentence does not convey any information.

Reply:
I will change it to `The reason may be you want to use an external serializer for only one object.`